### DISCUSSION TITLE
error upgrade to docusaurus 2.0.0-beta.4

### DISCUSSION THREAD

**Discussion by rafaelrdealmeida** [facebook/docusaurus#5413] [Q&A]
Hi, I'm trying to upgrade to docusaurus 2.0.0-beta.4.

when I try to run `yarn run build` I get the following error:

![Captura de tela de 2021-08-23 11-28-54](https://user-images.githubusercontent.com/18035347/130465547-8ad46ec9-6ac3-4f57-9136-791119f4e0fe.png)

already tried to remove the node_modules and clear the cache.

Any indication that how can I resolve this?

The site's repository is this: https://gitlab.com/unesp-labri/sites/labri

best

**Comment by Josh-Cena** [comment]
You don't have a `blog/` directory, so your first blog plugin can't find its source directory. The second and third do have their respective `cadernos/` and `noticias/` directories
